src/app/services/services-health.service.ts

```diff
@@ -11,15 +11,15 @@ export class ServicesHealthService {
   private GET_PING_URL = environment.BASE_URL + 'ping';
   private FOGLAMP_SHUTDOWN_URL = environment.BASE_URL + 'shutdown';
   private GET_SERVICES_URL = environment.BASE_URL + 'service';
+  private GET_PLUGINS_URL = environment.BASE_URL + 'plugins/installed';
```

Comment:
GET_INSTALLED_PLUGINS_URL, as later there will be another url which will allow to fetch available but not installed